cmd/manager/main.go

```diff
@@ -145,6 +145,7 @@ func init() {
 	viper.SetDefault(intctrlutil.FeatureGateEnableRuntimeMetrics, false)
 	viper.SetDefault(constant.CfgKBReconcileWorkers, 8)
 	viper.SetDefault(constant.FeatureGateIgnoreConfigTemplateDefaultMode, false)
+	viper.SetDefault(constant.FeatureGateDownwardAPIServiceDiscoveryMechanism, true)
```

Comment:
EnableComponentReplicasAnnotation is better, right ?

Reply:
fixed by [d84ab0f](https://github.com/apecloud/kubeblocks/pull/7902/commits/d84ab0f546716a7d804de1d2ca7feeddadb06ecd)